client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.scss

```diff
@@ -1,12 +1,10 @@
 .project-members__loading {
   font-size: 1em;
-  width: inherit;
+  justify-content: flex-end;
 }
 
 .project-members__cell {
   height: 31px;
-  display: flex;
-  align-items: center;
```

Comment:
I'm not sure if this is what's causing the issue, but the "R" in the "Remove" btn-link action to remove a project member is cut off
![Screen Shot 2021-05-12 at 5 29 01 PM](https://user-images.githubusercontent.com/20326896/118052246-03efda80-b348-11eb-868f-5ca7042458e9.png)

Reply:
Ooh. Good catch. (I cannot add/remove users locally.)

**Fixed.** 956cd31
![Fix Remove Button](https://user-images.githubusercontent.com/62723358/118188874-aa021a00-b406-11eb-887e-78520cd661ce.png)

Reply:
Seems to be happening again?
![Screen Shot 2021-05-24 at 6 07 18 PM](https://user-images.githubusercontent.com/20326896/119417439-6f786700-bcbb-11eb-96bc-09517388feb0.png)

Reply:
**Fixed.** 5a2f5d7 And in a way that will avoid the same bugs for other buttons in cells.

![Project Members Cell Button Bugs Fixed](https://user-images.githubusercontent.com/62723358/119542603-5cb27080-bd55-11eb-9dc1-2a2502ebff44.png)

_P.S. I have left font size unchanged for "Add Member" field/button and "Change Ownership" button link, because they are not in the table, and this table font size will cause unintended incongruences like that._